homeassistant/components/smlight/strings.json

```diff
@@ -19,7 +19,7 @@
       },
       "reauth_confirm": {
         "title": "[%key:common::config_flow::title::reauth%]",
-        "description": "Please enter the correct username and password for host: {host}",
+        "description": "Please enter the correct username and password for {name}",
```

Comment:
Given that name (host) is in the title now. lets just drop the host from this string.

`"description": "Please enter the correct username and password",`